www-servers/nginx-unit/nginx-unit-1.3.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="NGINX Unit is a dynamic web and application server."
+HOMEPAGE="https://unit.nginx.org"
+SRC_URI="https://unit.nginx.org/download/unit-${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="perl php python"
```

Comment:
how do these flags play? They do not alter users installation

Reply:
I'm sorry but I don't understand this comment and the next one (about dependencies). Could you elaborate? This is the first time I create an ebuild contribution, until now I have only used my ebuilds locally.

Reply:
If you were asking what the use- and dependency-flags do: Unit supports various application languages, and they need to be specified at compile-time so that the necessary helper modules are built and installed. For example, an emerge with `USE='perl python'` will create the following modules:

`/usr/modules/perl.unit.so`
`/usr/modules/python.unit.so`

The user needs to select at least one of the flags, because Unit does not build support modules unless it is told to do so.

Reply:
Yep, sorry, I missread this initially